src/main/java/com/unb/tracker/model/Course.java

```diff
@@ -14,9 +14,17 @@
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
     private Long id;
-    private String timeSlot;
+
```

Comment:
Does anyone know if these comments still need to be here? They were pulled in from a merge conflict

Reply:
I don't think so.... I left it there cause I wasn't sure